instrumentation/aerospike-client/aerospike-client-7.1/library/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id("otel.library-instrumentation")
```

Comment:
By library instrumentation we mean an instrumentation that can be used without javaagent which this module is not. In my opinion you should move the classes from this module to the javaagent module.

Reply:
Thanks for clarifying! I have moved the metrics to javaagent module.